services/web/server/src/simcore_service_webserver/projects/_crud_handlers.py

```diff
@@ -389,18 +389,13 @@ async def get_project_inactivity(request: web.Request):
     return web.json_response(Envelope(data=project_inactivity), dumps=json_dumps)
 
```

Comment:
Add the doc you removed inside `patch_project`  as a comment

```
# - Update https://google.aip.dev/134
```

Reply:
Same with the others ... in the middle, as they are now, they get misplaced with time .